docs/docusaurus/static/_redirects

```diff
@@ -447,4 +449,9 @@ http://legacy.docs.greatexpectations.io/* http://docs.greatexpectations.io 301!
 
 # abandoned pages
 /docs/0.18/oss/* /docs/0.18/core/introduction/introduction
-/docs/0.18/reference/learn/terms/* /docs/0.18/reference/learn/glossary
\ No newline at end of file
+/docs/0.18/reference/learn/terms/* /docs/0.18/reference/learn/glossary
+
+## Redirects for renamed resources docs
+
+/docs/resources/get_support /docs/help/get_support
+/docs/application_integration_support /docs/help/compatability_reference
```

Comment:
Expectation code has a huge number of links to the application integration support policy page and I don’t know why. At first I thought this was part of the docs string for the gallery, but I don't see these links showing up in the gallery. To keep this PR's scope down, I've left these links (several hundred of them) as they are to rely on this redirect. I asked about the purpose of the profusion of links here: https://greatexpectationslabs.slack.com/archives/C06MU0VTEB0/p1749755780697779